### PR TITLE
NML-10 Configure github workflow for synchronizing plugin apply example branch

### PR DESCRIPTION
**Pull request by gcwiak**

### CODE REVIEW

File: .github/workflows/syncPluginApplyExample.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Check repository
+      run: |
+        if [ "${{ github.event.pull_request.base.repo.html_url }}" != "${{ github.event.pull_request.head.repo.html_url }}" ]; then
+          echo "ERROR: Cross-repository pull requests, e.g. from forks, are not supported."
+          exit 1
+        fi
+    - name: Create sync pull request
```
Comment (j-bron) [inline .github/workflows/syncPluginApplyExample.yml]: One step is marked with _- uses_ and then _- name_. The other with _- name_ and then _uses_. Is there a convention on this or some meaning?

File: .github/workflows/syncPluginApplyExample.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      uses: repo-sync/pull-request@v2.0.1
+      with:
+        source_branch: "${{ github.event.pull_request.head.ref }}"
+        destination_branch: "master-with-nixer-plugin"
+        pr_title: "AUTO-SYNC: #${{ github.event.pull_request.number }} ${{ github.event.pull_request.title }}"
+        pr_body: ":leftwards_arrow_with_hook: **An automated sync PR**. Synchronizing `master-with-nixer-plugin` branch with changes on `master` introduced by PR ${{ github.event.pull_request.html_url }}."
+        pr_reviewer: "${{ github.event.pull_request.merged_by.login }}"
+        pr_assignee: "${{ github.event.pull_request.merged_by.login }}"
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (j-bron) [inline .github/workflows/syncPluginApplyExample.yml]: Where this token is defined in github?

Comment (gcwiak) [reply .github/workflows/syncPluginApplyExample.yml]: It's provided by GitHub.

Comment (gcwiak) [reply .github/workflows/syncPluginApplyExample.yml]: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token

File: .github/workflows/syncPluginApplyExample.yml (hunk 1 above)
Comment (gcwiak) [reply .github/workflows/syncPluginApplyExample.yml]: `name` field is not mandatory, so when omitted a step could contain just `- uses`. I added name to the checkout step to make it look more readable.

**Pull request merged**